recipe_scrapers/quitoque.py

```diff
@@ -0,0 +1,146 @@
+# mypy: allow-untyped-defs
+
+import requests
```

Comment:
```suggestion
import requests
```

A small cleanup opportunity here: this `mypy` comment can be removed.